data/plugins/language_python.lua

```diff
@@ -63,4 +42,95 @@ syntax.add {
     ["True"]     = "literal",
     ["False"]    = "literal",
   }
+
+
+local python_fstring = {
+
+    patterns = {
+      { pattern = {"{", "}"},  type = "normal", syntax = ".py" },
+      { pattern = "\\.",       type = "string"                 },
+      { pattern = '[^"\\{}]+', type = "string"                 }
```

Comment:
```suggestion
      { pattern = '[^"\\{]+',  type = "string"                 }
```
Matching `}` is probably desired here.